### ISSUE TITLE
lazy generator question

### ISSUE THREAD

**Issue by meowsynth** [HolographicWings/TOTK-Mods-collection#150]
question:
i generate lazy pack for 4k/60fps/nintendo ui  1.20 
what mods i need install else? (island fix? shadows..? LODs fix? fxaa fix? which ones are already included in lazy_pack?)
if i want cool stability, cool graphics, etc.
(i7 11700, rtx 3080)
im asking bcs im confused and its not clear which mods already come with the lazy pack and which dont.....

**Comment by hoverbike1** [comment]
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/57139770/35b6358b-f371-4642-962f-49b0d6928deb)
this information is available in both the release notes, and the description of the github page
https://github.com/HolographicWings/TOTK-Mods-collection

as always with lazy packs, you shouldn't mix them with other graphics mods - the lazy pack produces the optimal combination.
all of your suggestions are either redundant, or degrade the visual quality as they weren't meant to be used in conjunction with chuck's more recent iterations of DynamicFPS.

tldr; don't mix anything. its got you covered.
feel free to add whatever cheats and 'real' mods like, i dont know, bettersages?
if you use the lazy pack, you shouldnt use anything else from this repo.


except aspect ratios i guess. keep in mind that they share files with controller mods, so if you want to use both, you will have to merge the files in question using the developers own tool:
https://github.com/Fruithapje21/TOTK-Ultrawide

**Comment by meowsynth** [comment]
tysm, i little confused bcs i see "lod improvement 1.12", and think its not on 1.20, it hasnt been updated yet, i dont know))
love you <3
